Sourcery/CodeGenerated/Description.generated.swift

```diff
@@ -26,7 +26,7 @@ extension Enum.Case.AssociatedValue {
     override var description: String {
         var string = "\(type(of: self)): "
         string += "name = \(name), "
-        string += "type = \(type), "
+        string += "type = \(typeName), "
```

Comment:
This is what happens when people manually try to refactor code, instead of using Sourcery! :)